src/Shubhangi/assignment_4/Program_8_version_2.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * separate digits and characters from the user define string. 
+ * Using character method
+ */
+
+package Shubhangi.assignment_4;
+
+import java.util.Scanner;
+
+public class Program_8_version_2 {
+	String num="";
+	String alpha = "";
+	void separate(String str) {
+
+		StringBuffer alpha = new StringBuffer(),  
+		        num = new StringBuffer(), 
```

Comment:
When some operation on alpha or special that time it done on original and override the operation better to take String